test/packages/bad_discovery_fields/manifest.yml

```diff
@@ -0,0 +1,32 @@
+format_version: 3.4.0
+name: bad_discovery_fields
+title: "Bad Discovery Fields"
+version: 0.0.1
+source:
+  license: "Elastic-2.0"
+description: "Package to test discovery fields"
+type: content
+categories:
+  - custom
+conditions:
+  kibana:
+    version: "^8.18.3"
+  elastic:
+    subscription: "basic"
+discovery:
+  fields:
+    - name: 12345
+    - value: nginx.access
```

Comment:
Examples of incorrect discovery fields:
- `name` must be a string.
- a discovery field must contain a `name` field. `value` field is the optional one.